andela/travel_tool_front
# #160370620 Edit Users Trip Request

#### What does this PR do?
This PR contains the implementation for updating trip requests made by a user.
Trip requests could be a one way, return or multiple trip.

#### Description of Task to be completed?
- create reducers, action & sagas to edit a user trip request
- follow design mockup for edit tables
- write tests for editing a users trip request

#### How should this be manually tested?
- Clone this repo - `git clone https://github.com/andela/travel_tool_front.git`
- Switch to the repo directory - `cd travel_tool_front`
- Check out this branch - `git checkout ft-edit-trips-request-160370620 `
- Install dependencies - `yarn install`
- Start the server  - `make start`
- Navigate to `localhost:3000` in a browser
- Log in with your Andela mail
- Navigate to `my requests` pane
- Select the dotted buttons and click edit on the request you wish to make changes to


#### Any background context you want to provide?
- A request has to be made before one can think of editing, hence you have to first create a request before you can test out this feature.
- Accepted and rejected trip requests cannot be edited
- Managers cannot edit a request

#### What are the relevant pivotal tracker stories?
N/A


#### Screenshots (if appropriate)
<img width="1436" alt="screen shot 2018-09-27 at 4 52 08 pm" src="https://user-images.githubusercontent.com/22764463/46158467-ce208500-c275-11e8-80c5-3421d3b6f0e5.png">
<img width="1154" alt="screen shot 2018-09-27 at 4 52 32 pm" src="https://user-images.githubusercontent.com/22764463/46158476-d5479300-c275-11e8-883a-581c54a53de6.png">
<img width="1018" alt="screen shot 2018-09-27 at 4 53 38 pm" src="https://user-images.githubusercontent.com/22764463/46158516-ee504400-c275-11e8-873f-e59688d8e628.png">



#### Questions:
